src/main/webapp/app/shared/metis/forwarded-message/forwarded-message.component.ts

```diff
@@ -0,0 +1,114 @@
+import { AfterViewInit, ChangeDetectorRef, Component, ElementRef, OnChanges, OnInit, SimpleChanges, ViewChild, inject, input, output } from '@angular/core';
+import { faShare } from '@fortawesome/free-solid-svg-icons';
+import { Post } from 'app/entities/metis/post.model';
+import { AnswerPost } from 'app/entities/metis/answer-post.model';
+import { Posting } from 'app/entities/metis/posting.model';
+import dayjs from 'dayjs';
+import { Conversation } from 'app/entities/metis/conversation/conversation.model';
+
+@Component({
```

Comment:
Can we also make this standalone?

Reply:
it caused a circular dependency (with MetisModule), so I left it as it is for now